ChrstphrHll/316project
# Basic collections implementation

Basic implementation of the collections API and web pages.

Features to test:
1. url/collections
- Lists all collections from the database
2. url/collections/:cid
- Make sure correct games appear in a certain collection
- Make sure accurate data is displayed for each game. Will replace the games here with copies and their descriptions once we get that API implemented
3. url/collections/?search=:query OR use the search field in (1)
- Search by collection title
